dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/imports/validation/validator/event/SecurityOwnershipValidator.java

```diff
@@ -80,7 +79,10 @@ class SecurityOwnershipValidator implements Validator<org.hisp.dhis.tracker.impo
   @Nonnull private final OrganisationUnitService organisationUnitService;
 
   private static final String ORG_UNIT_NO_USER_ASSIGNED =
-      " has no organisation unit assigned, so we skip user validation";
+      "Event %s has no organisation unit assigned, so we skip user validation";
```

Comment:
when would an event have no org unit assigned? in case it's an event program?

Reply:
yes correct that is just a refactor to align with the new constant I've added